test/facility-locator/01-accessibility.e2e.spec.js

```diff
@@ -0,0 +1,57 @@
+const E2eHelpers = require('../e2e/e2e-helpers');
+const Timeouts = require('../e2e/timeouts.js');
+const FacilityHelpers = require('../e2e/facility-helpers');
+
+module.exports = E2eHelpers.createE2eTest(
+  (client) => {
+    client
+      .url(`${E2eHelpers.baseUrl}/facilities/`);
+
+    E2eHelpers.overrideSmoothScrolling(client);
+    FacilityHelpers.initApplicationMock();
+
+    client
+      .waitForElementVisible('body', Timeouts.normal)
+      .waitForElementVisible('.facility-locator', Timeouts.slow)
+      // do not run 'wcag2a' rules because of open aXe bug https://github.com/dequelabs/axe-core/issues/214
+      .axeCheck('.main', { rules: ['section508'] });
+
+    // Traverse form controls via keyboard input
+    client
+      .sendKeys('input[name="streetCityStateZip"]', client.Keys.TAB)
+      .execute(FacilityHelpers.checkActiveElement, ['div[class="facility-dropdown-wrapper"]'], result =>
```

Comment:
Can you make this a custom command or assertion? I haven't tried that for assertions, would be nice if that were possible.

Reply:
That would be much tidier, great idea.